pkg/cloudprovider/aws/aws.go

```diff
@@ -97,6 +184,217 @@ func (self *GoamzEC2) GetMetaData(key string) ([]byte, error) {
 	return v, nil
 }
 
+// Implements EC2.DescribeLoadBalancers
+func (self *goamzEC2) DescribeLoadBalancers(region string, findName string) (map[string]elb.LoadBalancer, error) {
+	client, err := self.getElbClient(region)
+	if err != nil {
+		return nil, err
+	}
+
+	request := &elb.DescribeLoadBalancer{}
+	// Names are limited to 32 characters, so we must use tags
+	//request.Names = []string{findName}
```

Comment:
It was supposed to indicate that we can't use names, but we would like to.  I will update the comment and remove the commented out code.